features/font-loading.yml

```
@@ -0,0 +1,45 @@
+name: CSS Font Loading
```

```suggestion
name: Font loading
```
The common name here is "CSS Font Loading API", which makes me grumpy as it's a JS API for loading fonts, which are used in CSS. I think we should either remove "CSS" or keep the whole name, including API.